src/Administration/Resources/app/administration/src/module/sw-order/component/sw-order-line-items-grid/sw-order-line-items-grid.html.twig

```diff
@@ -200,7 +200,7 @@
 
             {% block sw_order_line_items_grid_grid_columns_unit_price_inline_edit %}
             <mt-number-field
-                v-if="isInlineEdit && !itemCreatedFromProduct(item.id)"
+                v-if="isInlineEdit && !itemCreatedFromProduct(item.id) && item.priceDefinition"
```

Comment:
time for a variable? :grin:

Reply:
assuming you mean a computed property, I think in this case the `isInlineEdit` also needs to be passed to the method (so it is not a real variable), and maybe it is clearer in that way. So not really sure what will be gained with that change :-)